conf/dop/dop.yaml

```diff
@@ -39,7 +39,10 @@ erda.core.pipeline.cms-client:
 grpc-client@erda.core.pipeline.definition:
   addr: "${PIPELINE_GRPC_ADDR:pipeline:30810}"
 erda.core.pipeline.definition-client:
-erda.core.pipeline.definition-process-client:
```

Comment:
why just leave `definition-client` and remove `definition-process-client`? what difference are they?

Reply:
process was pipeline service client not grpc client, pipeline process service was removed